README.md

```diff
@@ -16,3 +16,26 @@ Create a `.env` file. See `.env.example` for environment variables which may be
 For Omen builds, use `yarn build`.
 
 The `build` directory in the `app` directory will now contain the build to be served.
+
+
+## Contracts
+
+| Contract | xDai | Sokol | Arbitrum |
+| -------- | -------- | -------- | -------- |
+| [ConditionalTokens](https://github.com/gnosis/conditional-tokens-contracts) | [0xCeAfDD6bc0bEF976fdCd1112955828E00543c0Ce](https://blockscout.com/poa/xdai/address/0xCeAfDD6bc0bEF976fdCd1112955828E00543c0Ce/read-contract)     | [0x0Db8C35045a830DC7F2A4dd87ef90e7A9Cd0534f](https://blockscout.com/poa/sokol/address/0x0Db8C35045a830DC7F2A4dd87ef90e7A9Cd0534f/contracts)     | -     |
+| FPMMDeterministicFactory | [0x9083A2B699c0a4AD06F63580BDE2635d26a3eeF0](https://blockscout.com/poa/xdai/address/0x9083A2B699c0a4AD06F63580BDE2635d26a3eeF0/contracts) | [0x2fb8cc057946DCFA32D8eA8115A1Dd630f6efea5](https://blockscout.com/poa/sokol/address/0x2fb8cc057946DCFA32D8eA8115A1Dd630f6efea5/contracts) | -     |
```

Comment:
We can add a link for the market maker contracts: https://github.com/gnosis/conditional-tokens-market-makers